src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -164,5 +170,32 @@ internal static Settings FromDefaultSources()
 
             return new Settings(configurationSource);
         }
+
+        private static OtlpExportProtocol GetExporterOtlpProtocol(IConfigurationSource source)
+        {
+            // the defualt in SDK is grpc. It is oposite that here.
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
+
+            if (exporterOtlpProtocol == "grpc")
+            {
+                return OtlpExportProtocol.Grpc;
+            }
+
+            return OtlpExportProtocol.HttpProtobuf;
+        }
+
+        private static Uri GetExporterOtlpEndpoint(IConfigurationSource source, OtlpExportProtocol otlpExportProtocol)
+        {
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpEndpoint);
```

Comment:
rename `exporterOtlpProtocol` to `exporterOtlpEndpoint` 😉

Reply:
fixed